sonar-project.properties

```diff
@@ -0,0 +1,15 @@
+sonar.projectKey=alphagov_govuk-mobile-ios-app
+sonar.projectName=govuk-mobile-ios-app
+sonar.pullrequest.github.repository=alphagov/govuk-mobile-ios-app
+sonar.projectVersion=0.1
```

Comment:
I think we might need to update the version in the project to match this... something we need to spec so its always matching, I think

Reply:
Done!